src/apis/stats_nba_com.py

```diff
@@ -0,0 +1,72 @@
+import json
+import selenium
+import sys
+from selenium import webdriver
+from time import sleep
+
+
+BROWSER_LOAD_DELAY = 6
+
+
+conn = None
+def get_connection():
```

Comment:
Every time you create a `StatsNbaComApi` object, it opens a new browser window.  This method exists to prevent you from having multiple browser windows open, by creating a global `StatsNbaComApi` instance and returning the same one every time.